client/app/components/faq/faqdetails.js

```diff
@@ -1,41 +1,27 @@
-import React, { Component } from 'react';
+import React from 'react';
 import {
-	AppRegistry,
 	Text,
 	View,
 } from 'react-native';
-import BottomNavigation, { Tab, NavigationComponent } from 'react-native-material-bottom-navigation'
-import Icon from 'react-native-vector-icons/Entypo'
-import { TabNavigator, StackNavigator, HeaderBackButton } from 'react-navigation'
 import { FaqStyles } from '../../styles/faqstyles'
 import { Header } from '../header'
 import { ListSlider } from '../listslider'
 
-class FaqDetails extends Component {
+class FaqDetails extends React.Component {
 	static navigationOptions = ({ navigation }) => ({
-		header: <Header
-					title={'Information'}
-					hasBackButton={true}
-					navigation={navigation}
-				  />
+		header: 
+			<Header
+				title="Information"
+				hasBackButton
+				navigation={navigation}
+			/>
 	});
 
 	constructor(props) {
 		super(props);
 	}
 
-	render() {
-		const { state } = this.props.navigation;
-		return (
-			<ListSlider
-				renderItem={this._renderItem}
-				currentIndex={state.params.index}
-				itemList={state.params.questionList}
-			/>
-		);
-	}
-
-	_renderItem(item, index) {
+	renderItem(item, index) {
```

Comment:
Go rid of the notation `_fooBar` because it doesn't make sense to do with React components. So `_fooBar` -> `fooBar`